### PR TITLE
issue [CHORE] add LinksHub contributors link in footer of the site #489

### LINKED ISSUE #489: [CHORE] add LinksHub contributors link in footer of the site
### What would you like to share or ask?

Add [this](https://github.com/rupali-codes/LinksHub/graphs/contributors) link in the footer of the site. The link should be open in **new tab**

### Additional information

image

### PR DESCRIPTION
**Pull request by 1010nishant**
## Fixes Issue
closes #489 

## Changes proposed
Text **open-source community** was not referring to any link. added mentioned link to it

## Screenshots
before
Screenshot 2023-04-29 at 12 04 42 PM
after
Screenshot 2023-04-29 at 12 05 03 PM

### CODE REVIEW

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -10,7 +10,15 @@ export const Footer: FC = () => {
...
           <span className="dark:text-gray-200 underline">Rupali Haldiya</span>
         </Link>{' '}
-        and <span className="dark:text-gray-200">open-source community</span>
+        and{' '}
+        <Link
+          href="https://github.com/rupali-codes/LinksHub/graphs/contributors"
+          target="_blank"
+        >
+          <span className="dark:text-gray-200 underline">
+            open-source community
```
Comment (rupali-codes) [inline components/Footer/Footer.tsx:19]:
```suggestion
           Open Source Community
```

**Review by rupali-codes — COMMENTED** [review]
Just a little suggestion, rest looks great to me, thanks :)

**Review by CBID2 — APPROVED** [review]
Looks great! :) LGTM! :)

**Comment by 1010nishant** [comment]
Thanks, @CBID2 for helping me

**Review by rupali-codes — APPROVED** [review]
Thanks, looks perfect

**Pull request merged**